src/analytics/analytics.php

```diff
@@ -105,6 +102,34 @@ private static function is_wpvip_site() {
 			&& defined( 'FILES_CLIENT_SITE_ID' )
 			&& function_exists( '\Automattic\VIP\Stats\send_pixel' );
 	}
+
+	/**
+	 * Check if the current request should be sampled for analytics.
+	 * 
+	 * Current sampling algorithm is that every 10s or 10m, we send analytics.
+	 * 
+	 * Max calls possible based on 1 call every 10s: 8640.
```

Comment:
@ingeniumed Please correct me if I'm wrong, but does this math work? We could have several requests come through in the same second which would result in multiple analytics calls in the same second. I think that's fine for sampling purposes, but it doesn't necessarily give us an upper limit on analytics requests.

Reply:
You are right, I added that in mainly for reference. But I don't want to make us think that we can use this to determine actual usage easily. I've taken it out